### ISSUE TITLE
Attacks incorrectly targeting Desert Eagles after they lock onto a unit.

### ISSUE THREAD

**Issue by Mmlh1** [cigumo/krvi#419]
https://cdn.discordapp.com/attachments/1172533591440371792/1193880589896519771/Falcon_Aim_1_-_Trim.mp4?ex=65ae52f1&is=659bddf1&hm=64b5b197d578c47e2e4f71de12ea2ef9bbfbe558d8fd4d885ef5a1015c57c3cc&

This is a pretty clear example, where the exploding head seems to use the position from where the Eagle locked onto Jack, rather than its current position. I believe more towers do this, so they will not target Eagles in your chokepoint most of the time as they count as being in their original position which is way further back, so towers will target all other enemies first since those are closer to the exit.

**Comment by Mmlh1** [comment]
This seems to already be fixed?

**Comment by pablorealini** [comment]
No, I fixed this for some cases, But cases like exploding head that target the middle of the road are still bugged.

**Comment by Mmlh1** [comment]
I noticed exploding head now targets the ground beneath Jack's feet. The explosion misses the Eagle, but at least he does not throw it at a completely different place.
